roxie/ccd/ccdserver.cpp

```diff
@@ -12404,8 +12399,8 @@ class CRoxieServerIndexWriteActivity : public CRoxieServerInternalSinkActivity,
 
     virtual void reset()
     {
-        noteStatistic(StNumDuplicateKeys, cummulativeDuplicateKeyCount);
         CRoxieServerActivity::reset();
+        noteStatistic(StNumDuplicateKeys, cummulativeDuplicateKeyCount);
```

Comment:
Not new but...

Feels like cumuativeDuplicateKeyCount should be cleared here? Though an index write in a child query is unlikely...

Also that's not how you spell cumulative

Reply:
I agree.  I'll push a separate fix for that.